knowledge-base/form-keep-fields-order.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Keep Form Items Rendering Order after Show/Hide
+description: How to keep the form item rendering order upon toggling its visibility.
+type: how-to
+page_title: Keep Form Items Rendering Order after Show/Hide
+slug: form-kb-keep-fields-order
+position: 
+tags: 
+ticketid: 1647539
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Form for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+This knowledge base article answers the following questions:
+
+* How do I keep the rendering order of Form items?
+* Is it possible to toggle the visibility of a Form item and keep its rendering order instead of showing it as last?
+
+## Solution
+
+The `<FormItems>` tag doesn't render Form items directly. This tag is used only to register `<FormItem>` instances. The Form finds out about the conditional field item last, so it puts it last.
```

Comment:
I feel that some context might be lacking here. What is the "conditional field item" mentioned at the end (I'm guessing this is the form item that gets shown/hidden and moves around the form)? You probably need to provide some extra explanation, such as "form items whose visibility is subject to change are considered conditional field items" (or whatever makes sense).